tests/dags/test_job_operator_jinja.py

```diff
@@ -0,0 +1,69 @@
+from utils import default_args
+from datetime import timedelta
+from airflow import DAG
+from airflow_kubernetes_job_operator import (
+    KubernetesJobOperator,
+    JobRunnerDeletePolicy,
+    KubernetesLegacyJobOperator,
+)
+
+dag = DAG(
+    "kub-job-op-test-jinja",
+    default_args=default_args,
+    description="Test base job operator",
+    schedule_interval=None,
+    catchup=False,
+    user_defined_macros={
+        "test_macro": lambda a: f"my {a}",
+        "default_image": "ubuntu",
+    },
+)
+
+namespace = None
+
+envs = {
+    "TIC_COUNT": 3,
+    "PASS_ARG": "a test",
+    "JINJA_ENV": "{{ ds }}",
+}
+
+default_delete_policy = JobRunnerDeletePolicy.Never
+
+KubernetesJobOperator(
+    task_id="test-job-success",
+    namespace=namespace,
+    image="{{default_image}}",
```

Comment:
Same here, why template the image itself? IMO the only part that needs to be able to use templates is the body.

Reply:
As I see it you should be able to use the operator without using a yaml body definitions. This would allow the image to be sent even through the xcom response from another task.

Reply:
True, I had a wrong representation of the whole templating thing. Learned a thing or two here :)